structures/multistep.js

```diff
@@ -0,0 +1,17 @@
+const s = require('superstruct')
+
+const Multistep = function (config = null) {
+  let MaxSteps = 50
+  if (config && config.shaq && config.shaq.maxsteps) MaxSteps = config.shaq.maxsteps
+
+  return s.size(s.array(s.string()), 38, 38*MaxSteps)
+}
+
+const PackageV1 = s.size(s.array(s.string()), 6)
```

Comment:
s.size(s.array(s.string()), 6) -> s.size(s.array(s.size(s.string(), 64)), 6) ? 

would be better to have s.tuple : string in first version but could be change later on.